worlds/stardew_valley/docs/setup_en.md

```diff
@@ -1,5 +1,13 @@
 # Stardew Valley Randomizer Setup Guide
 
+## Required Hardware
+- A computer
+- A keyboard or controller
+- A screen

```

Comment:
As a top level Stardew Valley speedrunner, you actually don't need a screen or keyboard/controller to beat the game. In my most recent challenge run, I beat Stardew Valley without even opening the game. So clearly, since everyone else is as good as I am at Stardew Valley, the keyboard, controller, and screen should be removed from the "Required Hardware" section.